src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -363,6 +372,37 @@ def get_command(self, additional_args=None):
     # list" and then run that fuzzer.
     return self.ssh_command('ls')
 
+  def fetch_and_process_logs_and_crash(self):
+    """Fetch symbolized logs and crashes."""
+    # Get the crash from the device.
+    self.device.fetch(
+        self.fuzzer.data_path('crash*'), self.fuzzer.results_output())
```

Comment:
this needs to take into account the other types of files such as oom- etc as well. Should we also do this in the parsing bits from line 390 onwards?